pkgs/applications/misc/orca/default.nix

```diff
@@ -0,0 +1,74 @@
+{ stdenv, lib, pkgconfig, fetchFromGitHub, pkgs
+, autoreconfHook, wrapGAppsHook
+, intltool, yelp-tools, itstool, libxmlxx3
+, python, pygobject3, gtk3
+, at_spi2_atk, at_spi2_core, pyatspi, dbus, dbus-python, pyxdg
+, louis ? null
+, xmodmap, xkbcomp, gsettings_desktop_schemas
+
+, speechdSupport ? true, speechd
+, brailleSupport ? false, brltty, brlapi ? null
+, setproctitleSupport ? true, setproctitle
+, gstreamerSupport ? true, gst_all_1, gst-python
+}:
+
+with lib;
+#pkgs.python3Packages.buildPythonPackage rec {
+stdenv.mkDerivation rec {
+  name = "orca-${version}";
+  version = "3.26.0";
+  # Tags in git are in the form of "ORCA_3_27_3"
+  gittag = concatStrings [ "ORCA_" (stringAsChars (x: if x == "." then "_" else x) version) ];
+
+  src = fetchFromGitHub {
+    owner = "GNOME";
+    repo = "orca";
+    rev = gittag;
+    sha256 = "0xpg7zgkp0kvcfpsfclrm3ssz4ryryfvk7vzbg8d18bdgw0zdw3b";
+  };
+
+  # doCheck = true;
+
+  nativeBuildInputs = [ autoreconfHook wrapGAppsHook pkgconfig libxmlxx3
+    intltool yelp-tools itstool ];
+
+  buildInputs = [ python pygobject3 gtk3 pyatspi dbus-python xkbcomp
```

Comment:
All the python libraries need to be in `propagatedBuildInputs`, see https://nixos.org/nixpkgs/manual/#handling-dependencies

Reply:
I suspect the manual is wrong there. I added this to issue  #13686.

Reply:
As vcunat mentions, unfortunately python derivations behave differently than `propagatedBuildInputs` is intended to be used.

Reply:
As it builds and runs fine, do you still want me to use the `propagetedBuldInputs`?

Reply:
You are right, adding `PYTHONPATH` to `wrapPrefixVariables` makes it work even without propagating the inputs. Though I would be wary of `PYTHONPATH` since it is passed to subprocesses. Not sure if it will be problem here, maybe @FRidh could clarify see also https://github.com/NixOS/nixpkgs/issues/22688